packages/x-components/src/x-modules/tagging/store/actions/track-tagging.action.ts

```diff
@@ -0,0 +1,42 @@
+import { TaggingInfo } from '../../../../../../search-types';
+import { XPlugin } from '../../../../plugins/x-plugin';
+import { DefaultSessionService } from '../../service';
+import { TaggingXStoreModule } from '../types';
+
```

Comment:
Please, update the `tagging/store/module.ts` initial state with `queryTaggingDebounceMs: 2000` instead of `200`